sw/device/lib/dif/dif_otbn_unittest.cc

```diff
@@ -321,6 +321,33 @@ TEST_F(GetErrBitsTest, Success) {
   EXPECT_EQ(err_bits, kDifOtbnErrBitsIllegalInsn | kDifOtbnErrBitsFatalReg);
 }
 
+class GetInsnCntTest : public OtbnTest {};
+
+TEST_F(GetInsnCntTest, NullArgs) {
+  dif_otbn_result_t result;
+  uint32_t insn_cnt;
+
+  result = dif_otbn_get_insn_cnt(nullptr, nullptr);
+  EXPECT_EQ(result, kDifOtbnBadArg);
+
+  result = dif_otbn_get_insn_cnt(&dif_otbn_, nullptr);
+  EXPECT_EQ(result, kDifOtbnBadArg);
+
+  insn_cnt = 55;
```

Comment:
You can declare the variable right here:

```suggestion
  uint32_t insn_cnt = 55;
```